packages/astro/test/fixtures/astro-collection/src/pages/$grouped.astro

```diff
@@ -1,30 +1,28 @@
 ---
-const { collection } = Astro.props;
 
-export async function createCollection() {
+export function createCollection() {
   const allPosts = Astro.fetchContent('./post/**/*.md');
   const allAuthors = allPosts.map(p => p.author);
   const uniqueAuthors = [...new Set(allAuthors)];
-
   return {
-    routes: uniqueAuthors.map(author => {
-        const params = { name: author };
-        return params;
-    }),
-
-    permalink: ({ params }) => `/grouped/${params.name}`,
-
-    async data({ params }) {
-        return allPosts.filter(p => p.author === params.name)
+    route: `/grouped/:name`,
```

Comment:
Would you mind also adding a test for a collection that's grouped and paged, something like `/grouped/:name/:page?`?